app/serializers/v1/payment_providers/customer_checkout_serializer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module V1
+  module PaymentProviders
+    class CustomerCheckoutSerializer < ModelSerializer
+      def serialize
+        {
+          lago_customer_id: model.id,
+          external_customer_id: model.external_id,
+          payment_provider: model.payment_provider,
+          checkout_url: options[:checkout_url],
```

Comment:
Maybe you could add an attribute to indicate witch payment provider is used for this checkout

Reply:
actually we have `payment_provider` field in the line 10:)